images/macos/tests/BasicTools.Tests.ps1

```diff
@@ -166,4 +166,10 @@ Describe "Homebrew" {
     It "Homebrew" {
         "brew --version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Kotlin" {
+    It "Kotlin" {
+        "kotlin -version" | Should -ReturnZeroExitCode
+    }
```

Comment:
The amount of tests for macOS and Ubuntu is different, should we add the similar tests here?